pymc/backends/arviz.py

```diff
@@ -63,6 +63,43 @@ def find_observations(model: Optional["Model"]) -> Optional[Dict[str, Var]]:
     return observations
 
 
+def find_constants(model: Optional["Model"]) -> Optional[Dict[str, Var]]:
+    """If there are constants available, return them as a dictionary."""
+    # For constant data, we are concerned only with deterministics and
+    # data. The constant data vars must be either pm.Data
+    # (TensorConstant/SharedVariable) or pm.Deterministic
+
+    if model is None:
+        return None
```

Comment:
Let's return the same type of variable to keep things simpler

```suggestion
        return {}
```

Reply:
Actually we should just assume a model is always passed. The caller shouldn't call `find_constants(None)`